composer.json

```diff
@@ -16,7 +16,8 @@
       "automattic/jetpack-connection": "1.20.0",
       "automattic/jetpack-config": "1.4.3",
       "automattic/jetpack-autoloader": "2.8.0",
-      "myclabs/php-enum": "1.7.7"
+      "myclabs/php-enum": "1.7.7",
+      "woocommerce/action-scheduler": "3.1.6"
```

Comment:
Let's add this dependency on `require-dev` instead so it doesn't end up in the plugin ZIP.